modules/cli/src/main/java/org/apache/ignite/internal/cli/util/ArrayUtils.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.internal.cli.util;
+
+/** Utility class for arrays. */
+public final class ArrayUtils {
+
+    private ArrayUtils() {
+    }
+
+    /** Finds the last not empty word in array. */
+    public static String findLastNotEmptyWord(String[] words) {
+        for (int i = words.length - 1; i >= 0; i--) {
```

Comment:
Same thing about boundaries checks and `isBlank`

Reply:
What's wrong with boundaries?

Reply:
oh, sorry, comment about boundaries is not applicable here